content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Gateway
+content_type: concept
+description: >-
+  Make network services available by using an extensible, role-oriented, protocol-aware
+  API. Gateway is a group of resources that provide dynamic infrastructure provisioning
+  and advanced traffic routing for Kubernetes services.
+weight: 55
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.29" state="stable" >}}
+
+Make network services available by using an extensible, role-oriented, protocol-aware configuration
+mechanism. The [Gateway][gw-api] API consists of [resources][resources] that provide dynamic
+infrastructure provisioning and advanced traffic routing.
+
+The Gateway project is maintained by the [SIG-NETWORK][sig-net] community outside of Kubernetes.
+Since certain Gateway API resources are considered stable, a subset of the full Gateway [documentation][gw-api]
+is included here.
+
+<!-- body -->
+
+## Concepts
+
+The following concepts form the foundation of Gateway:
+
+* __Role-oriented:__ Gateway API resources are modeled after organizational roles that are
+  responsible for managing Kubernetes service networking:
+  * __Infrastructure Provider:__ Manages infrastructure that allows multiple isolated clusters
+    to serve multiple tenants, e.g. a cloud provider.
+  * __Cluster Operator:__ Manages clusters and is typically concerned with policies, network
+    access, application permissions, etc.
+  * __Application Developer:__ Manages an application running in a cluster and is typically
+    concerned with application-level configuration and [Service][service] composition.
+* __Portable:__ Gateway API specifications are defined as [custom resources][crds] and are supported
+  by many [implementations][implementations].
+* __Expressive:__ Gateway API resources support functionality for common traffic routing use cases
+  such as header-based matching, traffic weighting, and others that were only possible in [Ingress][ingress]
+  by using custom annotations.
+* __Extensible:__ Gateway allows for custom resources to be linked at various layers of the API.
+  This makes granular customization possible at the appropriate places within the API structure.
```

Comment:
These are not concepts; they were the guiding principles to shape the design and architecture.